corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/form.html

```diff
@@ -47,7 +47,7 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
               visible: showSubmitButton,
               css: { 'sticky-submit': isAnchoredSubmitStyle },
               style: {
-                'bottom':isAnchoredSubmitStyle && {{ request.couch_user.can_edit_data|yesno:'true,false' }} {% if environment != "web-apps" %} && false {% endif %} ? '30px' : '0' {# data preview bar #}
+                'bottom':isAnchoredSubmitStyle && !isAppPreview && !smallScreenIsEnabled && {{ request.couch_user.can_edit_data|yesno:'true,false' }}? '30px' : '0' {# data preview bar #}
```

Comment:
I believe I did that and I also didn't prefer it. BTW this is only applicable for the sticky submit button (Where the submit button is shown as a bar anchored to the bottom of the screen) But from what I recall, it was tricky and involved some overhauling since the submit button and data preview bar and not within the same container. Also, both the anchored submit button and data preview bar are positioned using `position:fixed` and I couldn't figure out a way to make fixed items "float" even if they're on the same z-index.